admin/tools/docker/base/Dockerfile

```diff
@@ -196,6 +196,22 @@ RUN dnf install -y 'dnf-command(config-manager)' \
     && dnf clean all \
     && rm -rf /var/cache/yum
 
+ENV ARROW_VERSION 11.0.0
+
+RUN dnf update -y \
+    && dnf install -y \
+       https://apache.jfrog.io/artifactory/arrow/almalinux/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm \
+    && dnf config-manager --set-enabled epel \
+    && dnf config-manager --set-enabled powertools \
+    && dnf install -y arrow-devel-$ARROW_VERSION \
+    && dnf install -y arrow-dataset-devel-$ARROW_VERSION \
+    && dnf install -y arrow-flight-devel-$ARROW_VERSION \
+    && dnf install -y gandiva-devel-$ARROW_VERSION \
+    && dnf install -y parquet-devel-$ARROW_VERSION \
+    && dnf clean all \
+    && rm -rf /var/cache/yum
+
+
```

Comment:
I've noticed that all these packages were added here for the `lite-run-base` container. Meanwhile, my impression is that most of the packages are only needed for development purposes. For the development (compile, link, etc.) there is another container `lite-build`.  I'm also concerned that this extension may result in a significant increase in the size of the run-base container.

Is there a possibility to add the development tools to `lite-build` and the run-time tools (libraries, binaries) to `lite-run-base`?

@fritzm could you comment on this please?